setup.py

```diff
@@ -38,6 +38,7 @@
     "clu",  # All examples.
     "gym",
     "jaxlib",
+    "jraph==0.0.1.dev0",
```

Comment:
We try not to have explicit versions for the main `setup.py` but instead simply specify `jraph` for dependencies whenever possible. This simplifies maintenance.
(Note that existing explicit versions were added because without we would have had version conflicts; but I think `jraph` should work, please give it a try and update here if that's not the case)